common/types_spec.go

```diff
@@ -0,0 +1,256 @@
+package common
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/attestantio/go-builder-client/api"
+	"github.com/attestantio/go-builder-client/api/capella"
+	"github.com/attestantio/go-builder-client/spec"
+	consensusspec "github.com/attestantio/go-eth2-client/spec"
+	consensuscapella "github.com/attestantio/go-eth2-client/spec/capella"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	utilbellatrix "github.com/attestantio/go-eth2-client/util/bellatrix"
+	utilcapella "github.com/attestantio/go-eth2-client/util/capella"
+	"github.com/flashbots/go-boost-utils/bls"
+	boostTypes "github.com/flashbots/go-boost-utils/types"
+)
+
+var (
+	ErrMissingRequest     = errors.New("req is nil")
+	ErrMissingSecretKey   = errors.New("secret key is nil")
+	ErrInvalidTransaction = errors.New("invalid transaction")
+)
+
+type HTTPErrorResp struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+}
+
+var NilResponse = struct{}{}
+
+var VersionBellatrix boostTypes.VersionString = "bellatrix"
+
+var ZeroU256 = boostTypes.IntToU256(0)
+
+func BuildGetHeaderResponse(payload *BuilderSubmitBlockRequest, sk *bls.SecretKey, pubkey *boostTypes.PublicKey, domain boostTypes.Domain) (*GetHeaderResponse, error) {
+	if payload == nil {
+		return nil, ErrMissingRequest
+	}
+
+	if sk == nil {
+		return nil, ErrMissingSecretKey
+	}
+
+	if payload.Bellatrix != nil {
```

Comment:
dont need this anymore post capella, right?

Reply:
yes, we can remove all the bellatrix stuff from the relay and mev-boost codebase soon!